_posts/2020-11-17-quarkiverse.adoc

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: Quarkiverse Hub - Home of Community Extensions
+date: 2020-11-17
+tags: extensions quarkiverse
+synopsis: Entering a world of extensions
+author: gastaldi
+---
+
+Today we're proud to announce link:https://github.com/quarkiverse[Quarkiverse Hub], a GitHub organization where any contributor can host their Quarkus extensions.
+
+image:/assets/images/posts/quarkiverse/quarkiverse-repos.png[]
+
+Hosting your extension in the Quarkiverse Hub makes it available to all Quarkus users through the Quarkus tooling, and over time we'll start to add other features like a central extension registry.
```

Comment:
I'm not sure people will know what is meant by a "central extension registry". I presume you are talking about something like npmjs.com that provides pages like https://www.npmjs.com/package/ssh2 per extension? I don't know that we can call out that comparison directly. However, we could maybe describe the aspects of it that would interest our users. Thinking about some text to propose...